src/multiselectMixin.js

```diff
@@ -709,6 +709,24 @@ export default {
         this.preferredOpenDirection = 'above'
         this.optimizedHeight = Math.min(spaceAbove - 40, this.maxHeight)
       }
+    },
+    /**
+     * Clears the multiselect value/s
+     */
+    clearValue () {
+      /* istanbul ignore else */
+      if (this.disabled) return
+      /* istanbul ignore else */
+      if (!this.allowEmpty && this.internalValue.length <= 1) {
+        this.deactivate();
+        return
+      }
+      console.log('clear multiselect', this.internalValue);
```

Comment:
Thanks for the PR! QQ, do we want this `console.log` here?

Reply:
Ah, forgot to remove it, thanks.